dist/public/scripts/index.js

```diff
@@ -4,25 +4,23 @@
 
 displayUsers();
 
-
 function displayUsers() {
-    httpGet('/api/users/all')
-        .then(response => response.json())
-        .then((response) => {
-            var allUsers = response.users;
-            // Empty the anchor
-            var allUsersAnchor = document.getElementById('all-users-anchor');
-            allUsersAnchor.innerHTML = '';
-            // Append users to anchor
-            allUsers.forEach((user) => {
-                allUsersAnchor.innerHTML += getUserDisplayEle(user);
-            });
-        });
-};
-
+  httpGet("/api/users/all")
+    .then((response) => response.json())
+    .then((response) => {
+      var allUsers = response.users;
+      // Empty the anchor
+      var allUsersAnchor = document.getElementById("all-users-anchor");
```

Comment:
Is this something we eventually want to build into the React App?

Reply:
Yup, this was a vestige in`dist` and for some reason wasn't excluded from git

Reply:
Ah, `dist` was added to gitignore after these had already been checked in.

Reply:
[Deleted](https://github.com/Lambda-School-Labs/labs-be/pull/14/commits/1cb0f4360d5da765786db93f245a83f6547eaf1b)